packages/workbox-core/src/utils/assert.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * A simple assertion class to use for development builds.
+ *
+ * This will be excluded from production builds.
+ */
+class Assertions {
+
+}
+
+export default new Assertions();
```

Comment:
I wanted to solicit feedback from the larger group on the design choice of writing these sorts of modules as a class and then having the only export be a static instance of that class.

We do it a lot in the current code base and it's always struck me as, if not odd, something that I don't understand the motivation for.

What's the advantage vs. exporting a static object that had the same properties?